props/prop_base.h

```diff
@@ -547,12 +547,6 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     EnableBooster();
     BLADE_ID_CLASS_INTERNAL blade_id;
     float ret = blade_id.id();
-    PVLOG_STATUS << "BLADE ID: " << ret << "\n";
-#ifdef SPEAK_BLADE_ID
-    talkie.Say(spI);
-    talkie.Say(spD);
-    talkie.SayNumber((int)ret);
-#endif
 #ifdef BLADE_DETECT_PIN
     if (!blade_detected_) {
       STDOUT << "NO ";
```

Comment:
This printout (and the one on 561) needs to be moved to PrintAndSpeakId() as well.

Reply:
ok done see update